docs/user-guide/teacher-content/index.md

```diff
@@ -0,0 +1,22 @@
+# Teacher content
+
+VirtualShip is used as part of the VirtualShip Classroom, that combines authentic tools with VR to create a virtual fieldwork experience and allows you to teach about sea-based research from your regular classroom.
+
+All VirtualShip Classroom (VSC) material is open under an MIT licence and freely available! You can use the VSC to teach anything from a 4 hour masterclass up to an open assignment of more than 40 hours. Example assignments are available below and please feel free to customize anything offline or contribute to the assignments provided here. One of our core learning outcomes is for students to appreciate the difficulty of measuring the ocean. Additional Intended Learning Outcomes [(ILOs)](ILOs.md) were formulated in collaboration with ocean scientists from the Royal Netherlands Institute for Sea Research [(NIOZ)](https://www.nioz.nl/en) and Utrecht University [(UU)](https://www.uu.nl/en).
+
+The 360 videos are available on our YouTube channel [**_@VirtualShip Classroom_**](https://www.youtube.com/@VirtualShipClassroom) and in high resolution upon request. The videos can be viewed in the classroom or at home using:
+
+- VR Headsets – Wear a headset and look around naturally by moving your head.
+- Smartphones/Tablets – Move your device or swipe the screen to explore.
+- PC/Mac Browsers – Click and drag with your mouse to look around.
+
+The VSC design focuses on creating didactically sound, authentic learning experiences grounded in established learning theories in science education, such as constructivism (Piaget 1954) and constructionism (Papert 1980). By integrating realistic tasks and a gamified narrative approach within Jupyter notebooks, students learn within a digital replica of the real world, constructing knowledge through ‘learning by doing’ and ‘trial and error’ as they explore oceanography concepts, research methods, and analysis tools. We evaluated in several (under)graduate courses and find that the VirtualShip Classroom is highly engaging, and students report on enhanced self-efficacy and knowledge [(Daniels et al. 2025)](https://current-journal.com/articles/10.5334/cjme.121).
+
+```{nbgallery}
+---
+maxdepth: 1
+caption: Teaching material
+---
+
+ILOs.md
```

Comment:
Is it possible to add a picture to the tile? It now says "No Thumbnail"

<img width="229" height="261" alt="Image" src="https://github.com/user-attachments/assets/3a70c6b1-c96b-4e41-8275-aa26a90de25d" />

And while the asterisk is nice on the ILOs.md website itself, it may be a bit confusing here? Remove it?